packages/backend/src/tokens/request.ts

```diff
@@ -82,11 +82,7 @@ export type AuthenticateRequestOptions = RequiredVerifyTokenOptions &
   };
 
 export async function authenticateRequest(options: AuthenticateRequestOptions): Promise<RequestState> {
-  options.frontendApi =
-    (options as AuthenticateRequestOptionsWithExperimental).domain ||
-    parsePublishableKey(options.publishableKey)?.frontendApi ||
-    options.frontendApi ||
-    '';
+  options.frontendApi = parsePublishableKey(options.publishableKey)?.frontendApi || options.frontendApi || '';
```

Comment:
❓ the `domain` is removed? Are we sure about that?

Reply:
@dimkl This was intentional as we handle domain always separately and we should not expect the domain when reading frontendApi.